src/Workspaces/CSharp/Portable/Formatting/CSharpFormattingOptions.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.CSharp.Formatting
+{
+    public static partial class CSharpFormattingOptions
```

Comment:
I will file an issue to add doc comments on this type - there were no doc comments on the type or its public fields prior to this PR.

Reply:
💡  You can use `inheritdoc` to inherit the comments from `CSharpFormattingOptions2`, and then document them there.

Reply:
Sure, will do.

Reply:
I added inheritdoc and doc comments wherever possible, and filed https://github.com/dotnet/roslyn/issues/42522 for formatting options doc comments.